account/identity.go

```diff
@@ -312,6 +329,13 @@ func IdentityWithUser() func(db *gorm.DB) *gorm.DB {
 	}
 }
 
+// IdentithWithResource is a gorm filter for preloading the IdentityResource relationship
+func IdentityWithResource() func(db *gorm.DB) *gorm.DB {
+	return func(db *gorm.DB) *gorm.DB {
+		return db.Preload("IdentityResource")
```

Comment:
Need to check if this is a different query :)

Reply:
I've removed this function, gorm can't preload IdentityResource (and won't tell me why)